Source/Modules/php.cxx

```diff
@@ -714,7 +763,10 @@ class PHP : public Language {
 
     String * s = cs_entry;
     if (!s) s = s_entry;
-    Printf(s, " SWIG_ZEND_NAMED_FE(%(lower)s,%s,swig_arginfo_%s)\n", cname, iname, arginfo_code);
+    if (Cmp(cname,NULL) != 0)
```

Comment:
`Cmp()` against `NULL` here is a strange test (also used just above).  To test if `cname` is `NULL`, you want `if (!cname)` or `if (cname != NULL)` or similar.